src/lib/route-data/route-data-registry.ts

```diff
@@ -26,8 +26,15 @@ export class RouteDataRegistry {
     }
 
     public static getAll(): Readonly<RouteData> {
-        // TODO
-        return null;
+        const routeData: RouteData = Array.from(this.registry.entries()).reduce(
+            (accumulator: {}, [key, value]: [string, RouteDataInterface]) => ({
+                ...accumulator,
+                [key]: Object.freeze(value)
```

Comment:
Wie in meinen Anmerkungen beschrieben, wird die route data jetzt beim registrieren eingefroren (siehe https://github.com/plentymarkets/terra-components/pull/1482/files#diff-d224f9c0e7422c61d7d0bf234227a9dc2681e286d9c3fea4bbaf3fe9cda27110R42). Sprich hier wäre das nicht mehr notwendig.